qa/suites/upgrade-clients/client-upgrade-octopus-quincy/octopus-client-x/rbd/supported/ubuntu_20.04.yaml

```diff
@@ -0,0 +1 @@
+../../../../../../distros/all/ubuntu_18.04.yaml
```

Comment:
The symlink is named `ubuntu_20.04.yaml` but seems to points to `ubuntu_18.04.yaml`.  Is that intended?